INTMDB-299: Support Cloud Backup Export Jobs (#673)
mongodb/terraform-provider-mongodbatlas: website/docs/d/cloud_backup_snapshot_export_job.html.markdown

---
layout: "mongodbatlas"
page_title: "MongoDB Atlas: cloud_backup_snapshot_export_job"
sidebar_current: "docs-mongodbatlas-datasource-cloud_backup_snapshot_export_job"
description: |-
    Provides a Cloud Backup Snapshot Export Job resource.
---

# mongodbatlas_cloud_backup_snapshot_export_Job

> `mongodbatlas_cloud_backup_snapshot_export_job` datasource allows you to retrieve all the snapshot export jobs for the specified project and cluster.

Reviewer: ```suggestion
`mongodbatlas_cloud_backup_snapshot_export_job` datasource allows you to retrieve a snapshot export job for the specified project and cluster.
```

Author: done
